docker-compose.yml

```diff
@@ -13,11 +13,9 @@ services:
       - IEXEC_CORE_HOST=core
       - IEXEC_CORE_PORT=18090
       - IEXEC_WORKER_NAME=worker
-      - IEXEC_TEE_ENABLED:false
```

Comment:
Why do we remove this one? Is TEE enable by default if the machine supports it?

Reply:
Yes, that's what I thought unless we'd like to give the worker the choice to run TEE tasks?